tt_metal/common/work_split.cpp

```diff
@@ -148,6 +148,123 @@ CoreRangeSet num_cores_to_corerangeset(
     return num_cores_to_corerangeset({0, 0}, target_num_cores, grid_size, row_wise);
 }
 
+CoreRangeSet num_cores_to_corerangeset_in_subcoregrids(
+    const CoreCoord start_core,
+    const uint32_t target_num_cores,
+    const CoreRangeSet subcoregrids,
```

Comment:
Nit. Can we name as sub_core_grids to make it similar to naming of other args? Should also be const ref

Reply:
Done